amqp/src/main/java/io/github/tcdl/adapters/amqp/AmqpAdapter.java

```diff
@@ -75,7 +95,15 @@ public void unsubscribe() {
         }
     }
 
+    /**
+     * Represents the topic name for logging purpose
```

Comment:
This is not entirely correct description. We generate topic name in this way to get unique topics for different microservices. That's the main point of this code.